sdk/burdock/reader/sql/databricks.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+from pyspark.sql import Row

```

Comment:
We might be able to remove this dependency if we can use map reduce? udf might be a problem. Requiring spark for all scenarios won't be possible but we can require spark for this addition as a temporary solution. It may be worth having a burdock-spark package longer term since it looks like the spark ecosystem is perfect for DP.

Reply:
Agreed! Probably I can change it to return iterator in mapPartition function and remove this dependency. I'll try it.
As the first version, and it's working well, we can leave it as is for now.